cmd/action/action.go

```diff
@@ -0,0 +1,78 @@
+package action
```

Comment:
This package seems like it would be more appropriately place under `internal` -- `cmd` is really for the commands themselves.

Reply:
yeah, wasn't too sure where to place it - moved it to `internal/action`